website/docs/r/linux_web_app.html.markdown

```diff
@@ -0,0 +1,555 @@
+  Manages a Linux Web App.
+---
+
+# azurerm_linux_web_app
+
+Manages a Linux Web App.
+
+## Example Usage
+
+```hcl
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_service_plan" "example" {
+  name                = "example"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = "West Europe"
+  os_type             = "Linux"
+  sku_name            = "P1V2"
+}
+
+resource "azurerm_linux_web_app" "example" {
+  name                = "example"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_service_plan.example.location
+  service_plan_id     = azurerm_service_plan.example.id
+}
+
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `location` - (Required) The Azure Region where the Linux Web App should exist. Changing this forces a new Linux Web App to be created.
+
+* `name` - (Required) The name which should be used for this Linux Web App. Changing this forces a new Linux Web App to be created.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the Linux Web App should exist. Changing this forces a new Linux Web App to be created.
+
+* `service_plan_id` - (Required) The ID of the Service Plan that this Linux App Service will be created in.
+
+---
+
+* `app_settings` - (Optional) A map of key-value pairs of App Settings.
+
+* `auth_settings` - (Optional) A `auth_settings` block as defined below.
+
+* `backup` - (Optional) A `backup` block as defined below.
+
+* `client_affinity_enabled` - (Optional) Should Client Affinity be enabled?
+
+* `client_cert_enabled` - (Optional) Should Client Certificates be enabled?
+
+* `client_cert_mode` - (Optional) The Client Certificate mode. Possible values include `Optional` and `Required`. This property has no effect when `client_cert_enabled` is `false`
+
+* `connection_string` - (Optional) One or more `connection_string` blocks as defined below.
+
+* `enabled` - (Optional) Should the Linux Web App be enabled? Defaults to `true`.
+
+* `https_only` - (Optional) Should the Linux Web App require HTTPS connections.
+
+* `identity` - (Optional) A `identity` block as defined below.
+
+* `logs` - (Optional) A `logs` block as defined below.
+
+* `site_config` - (Optional) A `site_config` block as defined below.
+
+* `storage_account` - (Optional) One or more `storage_account` blocks as defined below.
+
+* `tags` - (Optional) A mapping of tags which should be assigned to the Linux Web App.
+
+---
+
+A `action` block supports the following:
+
+* `action_type` - (Required) Predefined action to be taken to an Auto Heal trigger. Possible values include: `Recycle`.
+
+* `minimum_process_execution_time` - (Optional) The minimum amount of time in `hh:mm:ss` the Linux Web App must have been running before the defined action will be run in the event of a trigger.
```

Comment:
this is a duration?
```suggestion
* `minimum_process_execution_duration` - (Optional) The minimum amount of time in `hh:mm:ss` the Linux Web App must have been running before the defined action will be run in the event of a trigger.
```

Reply:
I think time here is the typical nomenclature for this, rather than duration, as it's referring to something ongoing rather than something that is expected to end? I have no strong attachment, so happy to change it if we feel duration is more in keeping with the provider?